packages/story-editor/src/components/panels/shared/loopPanelContent.js

```diff
@@ -53,11 +54,13 @@ function LoopPanelContent({ loop, className = '', onChange, ...props }) {
         onChange={onChange}
         {...props}
       />
-      <label htmlFor={checkboxId}>
-        <Text as="span" size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
-          {__('Loop', 'web-stories')}
-        </Text>
-      </label>
+      <Text
```

Comment:
noticed this while working on some tests - the `Text` Styled Component can be a label, we should avoid putting a span with copy inside of a label.